source/develop/release-management/features/network/networking-api-security-groups.html.md

```diff
@@ -509,18 +516,52 @@ It will feature two ACLs, with the following data:
 + ingress rule
   * priority: DROP_PRIORITY
   * action: drop
-  * direction: from-lport
+  * direction: to-lport
   * match: inport == @<port_group> && ip
 + egress rule
   * priority: DROP_PRIORITY
   * action: drop
-  * direction: to-lport
+  * direction: from-lport
   * match: outport == @<port_group> && ip
 
 **NOTE:** these rules will be filtered out by the API, because they are simply
 not featured in the Networking API. This means that a user listing all security
 group rules **will not** see the rules corresponding to the ACLs defined above.
 
+### Default security group
+
+This group enables connectivity between all the VMs whose ports have this
+group set.
+
+To do so, it features 2 ingress rules - allowing traffic from all the VMs whose
+ports belong to the *Default* security group - and 2 egress rules - allowing
+traffic to leave the VM.
+Two rules are created for each direction: one for IPv4, another for IPv6.
+
+It features four ACLs, with the following data:
++ ingress rules
+  * rule #1 - IPv4
+    - priority: ALLOW_PRIORITY
+    - action: ALLOW
+    - direction: to-lport
+    - match: inport == @Default && ip4 && ip4.src == $pg_ip4_Default
+  * rule #1 - IPv6
```

Comment:
What is $pg_ip4_Default?

Reply:
that's the name of the address set where the IPs of the group member's IPs are stored.

I should probably mention that we rely on the remote_group_id parameter to implement this behavior.

Reply:
yes, I think this would be good

Reply:
@dominikholler Done